stories/topics/ref-aap-aws-uninstall.adoc

```diff
@@ -0,0 +1,35 @@
+[id="ref-aws-uninstall"]
+
+= Uninstall
+
+The following document guides how to uninstall {PlatformNameShort} from {Market}. If extension nodes were deployed, a prerequisite step must be taken to remove the extension nodes before uninstalling the stack.
+
+== Uninstall extension nodes
+
+If extension nodes were installed, follow these steps to remove them before continuing with the uninstall. If extension nodes were not installed, skip these steps.
+
+. In the {AWS} console, navigate to the link:https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#AutoScalingGroups[autoscaling groups page].
+.. Ensure you are in the same region as your stack.
+. Delete the auto scaling group for your extension nodes.
+.. This may be named *extension-group*.
+.. Select the checkbox next to the extension node auto scaling group.
+.. Click btn:[Delete].
+.. Confirm the deletion.
+. In the {AWS} console, navigate to the link:https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#LaunchTemplates[launch templates page].
+.. Ensure you are in the same region as your stack.
+. Delete the launch template for the extension nodes.
+.. This may be named *extension-group-template*.
+.. Select the launch template for your extension nodes.
+.. Click btn:[Actions] to drop down the available actions.
+.. Click btn:[Delete template].
+.. Confirm the deletion.
+. Extension nodes have been removed
+
+== Uninstall {PlatformNameShort} from {Market}
+
```

Comment:
.Procedure

Reply:
Added! I renamed this doc from ref to proc as well to reflect that. I can undo this as well if this was unnecessary